src/useSearchActions.ts

```diff
@@ -7,8 +7,8 @@ export type SearchActions = SearchHeadless;
 export function useSearchActions(): SearchActions {
   const searchHeadless = useContext(SearchHeadlessContext);
```

Comment:
do we want to update `SearchHeadlessContext` to accept an object of type `SearchActions` instead of `SearchHeadless`? right now, they are the same, which is why this works, but if we extended `SearchActions`, this code could be problematic